[2022-12-03 14:15:01,152] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.save_to_csv scheduled__2022-10-26T09:30:00+00:00 [queued]>
[2022-12-03 14:15:01,297] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.save_to_csv scheduled__2022-10-26T09:30:00+00:00 [queued]>
[2022-12-03 14:15:01,329] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 14:15:01,330] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-12-03 14:15:01,331] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 14:15:01,413] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): save_to_csv> on 2022-10-26 09:30:00+00:00
[2022-12-03 14:15:01,427] {standard_task_runner.py:52} INFO - Started process 687 to run task
[2022-12-03 14:15:01,463] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'save_to_csv', 'scheduled__2022-10-26T09:30:00+00:00', '--job-id', '214', '--raw', '--subdir', 'DAGS_FOLDER/AirflowClean2.py', '--cfg-path', '/tmp/tmp8m23j0qz', '--error-file', '/tmp/tmpylvjnc9b']
[2022-12-03 14:15:01,466] {standard_task_runner.py:77} INFO - Job 214: Subtask save_to_csv
[2022-12-03 14:15:01,789] {logging_mixin.py:109} INFO - Running <TaskInstance: dag1.save_to_csv scheduled__2022-10-26T09:30:00+00:00 [running]> on host 422c372562ce
[2022-12-03 14:15:02,080] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Fadlil
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=save_to_csv
AIRFLOW_CTX_EXECUTION_DATE=2022-10-26T09:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-26T09:30:00+00:00
[2022-12-03 14:15:02,359] {python.py:175} INFO - Done. Returned value was: None
[2022-12-03 14:15:02,525] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=save_to_csv, execution_date=20221026T093000, start_date=20221203T141501, end_date=20221203T141502
[2022-12-03 14:15:02,610] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-12-03 14:15:02,805] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
